cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
 | <<cypher-runtime, CYPHER runtime=compiled query>> | Runtime | [deprecated]#This will cause the query planner to use the compiled runtime if it supports `'query'`. This is only available in Neo4j Enterprise Edition.#
 |===
 
+[[glossary-admin-commands]]
+== Administrative commands
+All of these commands are only executable against the system database.
+
+[options="header"]
+|===
+|Command                                                                                                                                                                            | Category                      | Description
+| <<administration-databases-show-databases, SHOW DATABASE>>                                                                                                                        | Database administration       | Returns information about a specified database.
+| <<administration-databases-show-databases, SHOW DATABASES>>                                                                                                                       | Database administration       | Returns information about all databases.
+| <<administration-databases-show-databases, SHOW DEFAULT DATABASE>>                                                                                                                | Database administration       | Returns information about the default database.
+| <<administration-databases-create-database, CREATE [OR REPLACE] DATABASE ... [IF NOT EXISTS]>>                                                                                    | Database administration       | Crates a new database.
+| <<administration-databases-drop-database, DROP DATABASE ... [IF EXISTS]>>                                                                                                         | Database administration       | Deletes a specified database.
+| <<administration-databases-start-database, START DATABASE>>                                                                                                                       | Database administration       | Starts up a specified database.
+| <<administration-databases-stop-database, STOP DATABASE>>                                                                                                                         | Database administration       | Stops a specified database.
+| <<administration-security-roles-show, SHOW [ALL \| POPULATED] ROLES [WITH USERS]>>                                                                                                | User and role administration  | Returns information about all or populated roles, optionally including the assigned users.
+| <<administration-security-roles-create, CREATE [OR REPLACE] ROLE ... [IF NOT EXISTS] ... [AS COPY OF]>>                                                                           | User and role administration  | Creates new roles.
```

Comment:
There is nothing between `IF NOT EXISTS` and `AS COPY OF` so no need for the `...` there:
`CREATE [OR REPLACE] ROLE ... [IF NOT EXISTS] [AS COPY OF]`